deployment/aws/cdk/app.py

```diff
@@ -184,7 +184,7 @@ def __init__(
 if settings.mosaic_backend == "s3://" and settings.mosaic_host:
     perms.append(
         iam.PolicyStatement(
-            actions=["s3:GetObject", "s3:PutObject", "s3:HeadObject"],
+            actions=["s3:GetObject", "s3:PutObject"],
```

Comment:
s3:HeadObject isn't a permission, and I'd copied it from titiler permissions (which has also since removed it)